src/audio/mux/mux.c

```diff
@@ -101,8 +101,8 @@ static bool mux_mix_check(struct sof_mux_config *cfg)
 static int mux_set_values(struct comp_dev *dev, struct comp_data *cd,
 			  struct sof_mux_config *cfg)
 {
-	uint8_t i;
-	uint8_t j;
+	int i;
+	int j;
```

Comment:
Should this be "unsigned int i, j" ... considering "cfg->num_streams" is unsigned?

Reply:
unsigned int works as well, changing